### DISCUSSION TITLE
BLE solution hardware requirements

### DISCUSSION THREAD

**Discussion by benkw26** [RobertWojtowicz/export2garmin#49] [Q&A]
Based on my interpretation of the manual, I believe an additional bluetooth receiver adapter is required. Are there any recommend models / links to what they look like? Given the removable antenna is required, does this mean I cannot reconfigure bluetooth mice bluetooth receiver adapter?

**Comment by RobertWojtowicz** [comment]
Hi, @benkw26

There does not need to be an additional adapter, if you have an internal one, it can also work. It's a matter of the chipset whether it will handle it.

It does not have to be an outdoor antenna, however, it greatly improves range and use around the house.

Another aspect bluetotoh adapters can hang up and a separate adapter will not affect connected, for example, mouse.

I am using Zexmte BT5.1 on RTL8761B chipset: https://pl.aliexpress.com/item/1005006196126370.html

BR,
Robert
